go.mod

```diff
@@ -16,8 +15,8 @@ require (
 	github.com/pborman/uuid v1.2.1
 	github.com/robfig/cron v1.2.0
 	github.com/stretchr/objx v0.3.0 // indirect
-	github.com/stretchr/testify v1.7.1
-	go.temporal.io/api v1.8.0
+	github.com/stretchr/testify v1.7.2
```

Comment:
Usually when updating, we prefer a specific `go get go.temporal.io/api@v1.9.0` so it doesn't affect other deps. But this is fine (also that `make update-go-sum` target was unfortunately created using go get instead of go mod tidy).